pkg/cloud/api/check.go

```diff
@@ -171,3 +171,41 @@ func checkSchema(t reflect.Type) error {
 
 	return nil
 }
+
+// canConvertToT checks if we can convert type From to type To. Conversion is
+// possible if type From has the subset of fields with the same type and name as
+// in To type. This function is not symmetric.
+func canConvertToT(p Path, from, to reflect.Type) error {
```

Comment:
Should this be rename to something similar to `CheckConversion`? 

So the functions of `CheckXXX` would return an error, and `CanXXX` would return a boolean (e.g. `CanInterface`)

Reply:
done